### ISSUE TITLE
summarize_data failed with 'STOP' on all LJH files

### ISSUE THREAD

**Issue by joefowler** [usnistgov/mass#225]
**[Original report](https://bitbucket.org/joe_fowler/mass/issues/225) by Dan Becker (Bitbucket: [danbek](https://bitbucket.org/danbek), GitHub: [danbek](https://github.com/danbek)).**

----------------------------------------

I’m getting the following error with all the LJH files from one of our INL data runs:

```
channel.summarize_data chan 4096 100.0% done, estimated ? left
channel.summarize_data chan 4096 finished in 0.0 min
WARNING: Chan 4096 flagged bad because summarize_data failed with 'STOP'
```

The data was taken with dastard, but I don’t know the version number. I’m using mass 0.7.8 \(admittedly with a few modification of my own, but those mods are unrelated to summarizing data\).

I’ve attached a tgz file that should allow you to reproduce it.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Improve handling "auto" for excluded expt states

Fixes #225.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Merged in fix225 (pull request #184)

Fixes #225

Approved-by: Galen O'Neil

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Removing milestone: v0.7 (automated comment)
